pages/api/geojson/map-points.js

```diff
@@ -1,50 +1,30 @@
+// map-points show all properties that had violation in 2023
 import { PrismaClient } from '@prisma/client';
 
 const prisma = new PrismaClient();
 
 export default async function handler(req, res) {
   try {
+    // we use raw here because prisma doesn't have DISTINCT ON
     const results = await prisma.$queryRaw`
-      select
-        property."PID",
-        property."OWNER",
-        parcel."LOC_ID",
-        parcel."Shape_STLength__",
-        sam."FULL_ADDRESS",
-        sam."MAILING_NEIGHBORHOOD",
-        sam."ZIP_CODE",
-        bpv."code",
-        bpv."description",
+      select DISTINCT ON (bpv."sam_id")
         bpv."latitude",
-        bpv."longitude"
-      from property property
-      join parcel parcel on property."PID" = parcel."MAP_PAR_ID"
-      join sam sam on property."PID" = sam."PARCEL"
-      join bpv bpv on sam."SAM_ADDRESS_ID" = bpv."sam_id"
-      where cast(property."year" as integer) = 2023;`;
+        bpv."longitude",
+        bpv."sam_id"
+      from bpv bpv
+      where bpv."status_dttm" > '2023-01-01 00:00:00'`;
```

Comment:
You should restrict this to just 2023, planning for the new year as this would break.